bot/__main__.py

```diff
@@ -27,7 +27,7 @@
     "bot.cogs.embeds",
     "bot.cogs.comics",
     "bot.cogs.coding",
-    "bot.cogs.documentation"
+    "bot.cogs.reddit",
```

Comment:
Why is this removing documentation and replacing it with reddit, reddit should be a new cog but there is no reason to remove documentation cog.